sirepo/package_data/static/js/cloudmc.js

```diff
@@ -2110,7 +2085,115 @@ SIREPO.app.directive('materialList', function(appState, cloudmcService) {
     };
 });
 
+SIREPO.app.directive('planePositionSlider', function(appState, tallyService) {
+    return {
+        restrict: 'A',
+        scope: {},
+        template: `
+            <div data-ng-show="hasSteps()">
```

Comment:
I think this case is confusing because it was showing a slider which couldn't be used - the low/high value were the same and if you try to manipulate it, it isn't clear what it does. The 0 position is already shown in the report title in this case.
![x](https://github.com/radiasoft/sirepo/assets/6516910/eef28597-fa48-4043-b755-acebcc5d4df3)